arm/Microsoft.Network/expressRouteCircuits/deploy.bicep

```diff
@@ -205,4 +205,4 @@ output expressRouteCircuitResourceGroup string = resourceGroup().name
 output expressRouteCircuitName string = expressRouteCircuits.name
 
 @description('The service key of the express route circuit')
-output expressRouteCircuitServiceKey string = reference(expressRouteCircuits.id, '2020-05-01').serviceKey
+output expressRouteCircuitServiceKey string = reference(expressRouteCircuits.id, '2017-04-01').serviceKey
```

Comment:
Was that intended?

Reply:
No. Will fix it

Reply:
Please re-run the pipeline when upating this to a new API version to be sure it works

Reply:
[![Network: ExpressRouteCircuits](https://github.com/Azure/ResourceModules/actions/workflows/ms.network.expressroutecircuits.yml/badge.svg)](https://github.com/Azure/ResourceModules/actions/workflows/ms.network.expressroutecircuits.yml)